src/index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Colored-blocks</title>
+</head>
+<body>
+<div class="redColor blockHeight"></div>
```

Comment:
don't name class with camelCase

Reply:
In this task ok, but for the future, `redColor, blockHeight...` bad name for the class.